### PR TITLE
done HW lesson 17

### PR DESCRIPTION
**Pull request by DergachovaAnna**
done hw for lesson 17

### CODE REVIEW

**Comment by bohdandovbsyh** [comment]
Where s requirenments.txt?

File: lesson_17/tests/conftest.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+from lesson_17.Human.human import Human
+
+
+@pytest.fixture
+def create_human_positive():
+    return Human("Alice", 16, 'female')
+
+
+@pytest.fixture
+def create_human_negative():
```
Comment (bohdandovbsyh) [inline lesson_17/tests/conftest.py]: whad does it mean negative human)) 
please rename

File: lesson_17/tests/conftest.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import pytest
+from lesson_17.Human.human import Human
+
+
+@pytest.fixture
+def create_human_positive():
```
Comment (bohdandovbsyh) [inline lesson_17/tests/conftest.py]: what does it mean positive human))
please rename

File: lesson_17/tests/test_human.py [hunk 3]
```diff
@@ -0,0 +1,99 @@
+import pytest
+from lesson_17.Human.human import Human
+
+
+@pytest.mark.parametrize("name,age,gender", [('Ann', 1, "female"), ('Bob', 99, "male")])
+@pytest.mark.positive
+def test_create_human(name, age, gender):
+    human = Human(name, age, gender)
```
Comment (bohdandovbsyh) [inline lesson_17/tests/test_human.py:8]: you should not create Human directly in the test. much better to implement such fixture

File: lesson_17/tests/test_human.py [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+def test_change_to_existed_gender(create_human_positive):
+    # Try to change to the same gender
+    new_gender = "male" if create_human_positive.gender == "male" else "female"
+    with pytest.raises(Exception):
+        create_human_positive.change_gender(new_gender)
+
+
+@pytest.mark.positive
+def test_grow_human_below_limit(create_human_positive):
+    for number in range(create_human_positive.age, 99):
```
Comment (bohdandovbsyh) [inline lesson_17/tests/test_human.py]: I do not think that verifying all available ages is a good idea. 
Just use Boundary Value Analysis

**Review by bohdandovbsyh — APPROVED** [review]
